volatility3/framework/plugins/windows/callbacks.py

```diff
@@ -367,12 +640,16 @@ def list_bugcheck_callbacks(
             except exceptions.InvalidAddressException:
                 component = renderers.UnreadableValue()
 
-            yield "KeBugCheckCallbackListHead", callback.CallbackRoutine, component
+            yield (
+                "KeBugCheckCallbackListHead",
+                cast(int, callback.CallbackRoutine),
+                component,
+            )
 
     def _generator(self):
-        kernel = self.context.modules[self.config["kernel"]]
+        kernel = self.context.modules[cast(str, self.config["kernel"])]
```

Comment:
I'm interested to know under what circumstances this wouldn't be seen as a string by the type checker?  I'm all for improving the typing, but I'd prefer it not get out of hand manually casting everything rather than getting the types right at the source?

Reply:
I was wondering if I was getting carried away with the type-casting - I was trying to make things as explicit as possible but this may have been overkill, so I'll roll that change back.